ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.8__4.3.yaml

```diff
@@ -0,0 +1,45 @@
+base_images:
+  base:
+    name: "4.3"
+    namespace: ocp
+    tag: base
+build_root:
+  project_image:
+    dockerfile_path: openshift/ci-operator/build-image/Dockerfile
+images:
+- context_dir: serving/ingress
+  from: base
+  to: knative-openshift-ingress
+- context_dir: knative-operator
+  from: base
+  to: knative-operator
+promotion:
+  name: openshift-serverless-v1.8.0-rc1
+  namespace: openshift
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.3"
+  namespace: ocp
+tests:
+- as: e2e-aws-ocp-43
```

Comment:
We now use a little different approach on master branch, see https://github.com/openshift/release/blob/master/ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-master__4.3.yaml#L56 
It's the multi-step tests. 
This one should be merged first: https://github.com/openshift/release/pull/9685 . And then you'll be able to see the final/minimal format that works.

Reply:
It also needs the serverless-operator-src image that is defined here: https://github.com/openshift/release/blob/master/ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-master__4.3.yaml#L22

Reply:
Same applies to 4.4 and 4.5 branches. The rest of PR looks good.

Reply:
This is required because we now include sources of Serving and Eventing at ci-operator level. We don't have the "clones" in the repo so the previous approach will be failing tests.

Reply:
Ok, the other PR is failing ... wanna PR some change to my branch? So we sort it out for 1.8 ?

Reply:
I've just noticed your comment. Will send something in a while.